thanks/index.md

```diff
@@ -0,0 +1,166 @@
+* Alec Ross, Arcs Ltd.
+* Sandro Sigala, Roxantis
+
+[Seventh String Software]: https://www.seventhstring.com/
+
+### Hosting
+
+The stipends from our participation in [Google Summer of Code][GSoC] allow Bryan Petty to finance the hosting for this website, the online user manuals, wxTrac, the read-only SVN and the wxWiki. 
+
+[GSoC]: https://summerofcode.withgoogle.com
+
+## Credits
+
+There are hundreds of people who have made wxWidgets possible through donated
+hardware, services, development, documentation, translations, and support. We
+couldn't possibly list them all here over the last twenty years, however, we
+would like to highlight just a few of those rockstars.
+
+### Developers
+
+We would particularly like to thank the following for their [contributions to
+wxWidgets][github contributors], and the many others who have been involved in the project over the
+years. Apologies for any unintentional omissions from this list.
+
+[github contributors]: https://github.com/wxWidgets/wxWidgets/graphs/contributors
+
+#### wxWidgets 3
+
+Contributors (with several commits)
+
+Vadim Zeitlin, Artur Wieczorek, Paul Cornett, Maarten Bent, Tobias Taschner, Stefan Csomor, Cătălin Răceanu, Václav Slavík, Robin Dunn, Dimitri Schoolwerth, Graham Dawes, PB, New Pagodi, Ilya Sinitsyn, Jouk Jansen, Lauri Nurmi, Ian McInerney, Jay Nabonne, Julian Smart, Scott Talbert, ARATA Mizuki, Igor Korot, Tim Stahlhut, Pavel Tyunin, Andreas Falkenhahn, Liam Treacy, Bryan Petty, AliKet, Hartwig Wiesmann, Paul Kulchenko, Daniel Kulp, Steve Browne, Iwbnwif Yiw, John Roberts, Matthew Griffin, Sean D'Epagnier, Pavel Kalugin, Danny Scott, Arrigo Marchiori, Martin Ettl, Richard Smith, Troels Knakkergaard, Jorge Moraleda, Kvaz1r, Deamhan, Tim Kosse, Hertatijanto Hartono, Kolya Kosenko, Stefan Brüns, Olly Betts, Kevin B. McCarty, wangqr, Scott Furry, Tomay, Anton Triest, Blake Eryx, chris2oph, Samuel Dunn, Gary Allen, Rafael Kitover, Konstantin S. Matveyev, mimi89999, Jens Göpfert, Jeff Bland, Priit Laes, Frédéric Bron, Raul Tambre, Pavel O., dsr, Kinaou Hervé, Teodor Petrov, NikitaFeodonit, Marc-Philip, Markus Juergens, Martin Srebotnjak, Max Maisel, jensgoe, Xaviou, Roberto Perpuly, René Kijewski, Artur Sochirca
+
+#### wxWidgets 2 and 1
+
+Main Authors:
+
+* wxWidgets 2 for GTK+: Robert Roebling - Latin scholar, medic, Renaissance man
+* wxWidgets 2 for Windows: Julian Smart - Multum in Parvo in Rutland
+* wxWidgets 2 for Motif: Julian Smart - didn't quite mean to do that port too,
+  oh well
+* wxWidgets 2 for Mac: Stefan Csomor - keeping baby/keyboard separation optimal
+* wxWidgets 1 for Xt: Markus Holzem - keeping his sanity now he's away
+* wxString, wxConfig, Windows drag and drop, and loads of other great stuff
+  without which we would be quite stuck: the one and only Vadim Zeitlin,
+  physicist and wxWidgets hero
+* wxThread, wxSocket and multimedia classes: the deceptively young but talented
+  Guilhem Lavaux
+* wxHTML: the highly dedicated Vaclav Slavik
```

Comment:
Another nitpick: Václav's name is listed in wxWidgets 3 contributors with diacritics, and he uses diacritics not only on GitHub but also on his English-written web homepage. Perhaps the same form of his name should be used everywhere in this document.
```suggestion
* wxHTML: the highly dedicated Václav Slavík
```

Reply:
Thanks